doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1590,6 +1590,9 @@ Binding types or coercion classes to notation scopes
    then :g:`a` of type :g:`t` in :g:`f t a` is not recognized as an argument to
    be interpreted in scope ``scope``.
 
+   In explicit casts :g:`term : coercion_class`, the :g:`term`
+   will also be interpreted in :token:`scope_name`.
+
```

Comment:
If this applies to all casts (not just coercion_classes), then this should also be mentioned where `:>` is defined, i.e. https://coq.inria.fr/refman/language/core/definitions.html#type-cast

Reply:
Good point, done.